strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/storage/checksum/ChecksumCacheManager.java

```diff
@@ -27,200 +36,97 @@
  * remove the respective Checksum from the cachedChecksums).
  *
  * @author mtodorov
+ * @author Przemysław Fusik
  */
+@Component
 public class ChecksumCacheManager
+        implements DisposableBean
 {
 
-    private static Logger logger = LoggerFactory.getLogger(ChecksumCacheManager.class);
+    private static final String SHA1_KEY = "sha1";
 
-    /**
-     * Key:     Artifact path
-     * Value:   Artifact checksum.
-     */
-    private Map<String, ArtifactChecksum> cachedChecksums = new LinkedHashMap<>();
+    private static final String MD5_KEY = "md5";
```

Comment:
Hm... Could you perhaps use something like `EncryptionAlgorithmsEnum.MD5.getAlgorithm()` instead?